README.md

```diff
@@ -98,13 +98,14 @@ Please see the following resources for more information on MediaStreamConstraint
 
 ### DTMF
 
-- Description: send a single VoIP-network friendly DTMF tone. The tone amplitude and duration can not be controlled.
+- Description: send a set of VoIP-network-friendly DTMFs tones. The tone amplitude and duration can not be controlled.
```

Comment:
Sorry to be nit-picky but I don't think DMTF should be plural

Reply:
Not a nit-pick at all.  I am completely baffled at how / why I did that.